docs/source/platform/client-awareness.md

```diff
@@ -1,4 +1,117 @@
 ---
 title: Identifying clients
-description: How to configure Apollo Engine for client awareness
----
\ No newline at end of file
+description: What is client awareness and how to add it to the Apollo Platform
+---
+
+Client identity is central to the Apollo Platform and enables tracking how all
+the consumers use the data graph. The Apollo Platform allows segmenting usage
+data by client name and version. Filtering by client information provides a
+field-level understanding of how the users interact with the GraphQL api in
+real-time. In addition to per client metrics, understanding this granular
+detail informs how the GraphQL schema can evolve and reacts to a new client
+release.
+
+## Use Cases
+
+Often a GraphQL api is used by multiple consumers with different frequencies,
+subselections, and permissions. The Apollo Platform accepts client name and
+version, tracking these requests across the stack and enabling filtering on
+both dimensions. This segmentation provides an understanding of which queries
+and fields are required by clients. Conversely, this breakdown also indicates
+which clients are important based on relative usage.
+
+![client overview](../img/client-awareness/overview.png)
+
+In addition to understanding how clients use a GraphQL api, the Apollo Platform
+enables effective isolation and rapid triage for issues that affect a single
+client or portion of clients begin to experience issues, by filtering out of
```

Comment:
Something is off with this paragraph. 

> ...that affect a single client or portion of clients begin to experience issues,..

Please rephrase

Reply:
Agreed -- this sentence doesn't sound quite right. What are you trying to say here?